### DISCUSSION TITLE
Start Fooocus with advanced menu enabled

### DISCUSSION THREAD

**Discussion by optotechgr** [lllyasviel/Fooocus#3316] [Q&A] [answered]
Hi, is it possible to start fooocus with the advanced menu ticked (enabled) by default?

**Comment by mashb1t** [comment, marked as answer]
Yes, you can set `default_advanced_checkbox: True` in the config.txt. Please find even more possible configurations in the comfig example file.
Basically everything in config.pycan be set using the config, see https://github.com/lllyasviel/Fooocus/blob/main/modules/config.py#L406

**Comment by optotechgr** [comment]
Hi mashb1t, thank you for your prompt answer. It didn't work though. I know that it should but it didn't. Should I changed something else also?

default_advanced_checkbox = get_config_item_or_set_default(
    key='default_advanced_checkbox',
    default_value=True,
    validator=lambda x: isinstance(x, bool),
    expected_type=bool
)

**Comment by mashb1t** [comment]
> Yes, you can set default_advanced_checkbox: True in the config.txt.

Do not change the code but adjust the config.txt instead following the example shown in the config example file.

**Reply by optotechgr** [reply]
Thanks!!! Works like a charm!
